strings/base_stringable_format.h

```diff
@@ -0,0 +1,12 @@
+
+#ifdef __cpp_lib_format
+template<>
+struct std::formatter<winrt::Windows::Foundation::IStringable, wchar_t> : std::formatter<winrt::hstring, wchar_t>
+{
+    template<typename FormatContext>
+    auto format(winrt::Windows::Foundation::IStringable const& obj, FormatContext& fc)
+    {
+        return std::formatter<winrt::hstring, wchar_t>::format(obj.ToString(), fc);
+    }
+};
```

Comment:
It is unfortunate that you cannot SFINAE a partial specialization.

Reply:
Every day I wonder why I'm still using C++ 😂

Reply:
This is why I'm enjoying Rust these days. 😎